tests/unittests/nominal/test_cramers.py

```diff
@@ -0,0 +1,166 @@
+        cramers_v_matrix_value[i, j] = cramers_v_matrix_value[j, i] = _dython_cramers_v(
+            x, y, bias_correction, nan_strategy, nan_replace_value
+        )
+    return cramers_v_matrix_value
+
+
+@pytest.mark.skipif(not _DYTHON_AVAILABLE, reason="test requires dython")
+@pytest.mark.parametrize(
+    "preds, target",
+    [
+        (_input_default.preds, _input_default.target),
+        (_input_with_nans.preds, _input_with_nans.target),
+        (_input_logits.preds, _input_logits.target),
+    ],
+)
+@pytest.mark.parametrize("bias_correction", [False, True])
+@pytest.mark.parametrize("nan_strategy, nan_replace_value", [("replace", 0.0), ("drop", None)])
+class TestCramersV(MetricTester):
+    @pytest.mark.parametrize("ddp", [False, True])
+    @pytest.mark.parametrize("dist_sync_on_step", [False, True])
+    def test_cramers_v(self, ddp, dist_sync_on_step, preds, target, bias_correction, nan_strategy, nan_replace_value):
+        metric_args = {
+            "bias_correction": bias_correction,
+            "nan_strategy": nan_strategy,
+            "nan_replace_value": nan_replace_value,
+            "num_classes": NUM_CLASSES,
+        }
+        reference_metric = partial(
+            _dython_cramers_v,
+            bias_correction=bias_correction,
+            nan_strategy=nan_strategy,
+            nan_replace_value=nan_replace_value,
+        )
+        self.run_class_metric_test(
+            ddp=ddp,
+            dist_sync_on_step=dist_sync_on_step,
+            preds=preds,
+            target=target,
+            metric_class=CramersV,
+            sk_metric=reference_metric,
+            metric_args=metric_args,
+        )
+
+    def test_cramers_v_functional(self, preds, target, bias_correction, nan_strategy, nan_replace_value):
+        metric_args = {
+            "bias_correction": bias_correction,
+            "nan_strategy": nan_strategy,
+            "nan_replace_value": nan_replace_value,
+        }
+        reference_metric = partial(
+            _dython_cramers_v,
+            bias_correction=bias_correction,
+            nan_strategy=nan_strategy,
+            nan_replace_value=nan_replace_value,
+        )
+        self.run_functional_metric_test(
+            preds, target, metric_functional=cramers_v, sk_metric=reference_metric, metric_args=metric_args
+        )
+
+    def test_cramers_v_differentiability(self, preds, target, bias_correction, nan_strategy, nan_replace_value):
+        metric_args = {
+            "bias_correction": bias_correction,
+            "nan_strategy": nan_strategy,
+            "nan_replace_value": nan_replace_value,
+            "num_classes": NUM_CLASSES,
+        }
+        self.run_differentiability_test(
+            preds,
+            target,
+            metric_module=CramersV,
+            metric_functional=cramers_v,
+            metric_args=metric_args,
+        )
+
+
+@pytest.mark.skipif(not _DYTHON_AVAILABLE, reason="test requires dython")
```

Comment:
why is this needed? :/

Reply:
`dython` is a reference package installed only with `test` extras.

Reply:
That seems rather as bug in CI